src/applications/personalization/dashboard-2/tests/unit/healthcare-appointments.unit.spec.jsx

```diff
@@ -0,0 +1,178 @@
+import React from 'react';
+import { expect } from 'chai';
+import { renderInReduxProvider } from '~/platform/testing/unit/react-testing-library-helpers';
+import reducers from '~/applications/personalization/dashboard/reducers';
+import HealthCare from '~/applications/personalization/dashboard-2/components/health-care/HealthCare';
+import {
+  farFutureAppointments,
+  upcomingVAAppointment,
+} from '~/applications/personalization/dashboard-2/utils/appointments';
+
+describe('HealthCare component', () => {
+  let view;
+  let initialState;
+
+  describe('when the user has an appointment scheduled within the next 30 days', () => {
+    it('should render "Next appointment"', async () => {
+      window.VetsGov = { pollTimeout: 1 };
+      initialState = {
+        user: {
+          profile: {
+            services: [],
+          },
+        },
+        health: {
+          appointments: {
+            fetching: false,
+            data: upcomingVAAppointment,
+          },
+          msg: {
+            folders: {
+              data: {
+                currentItem: {
+                  attributes: {
+                    unreadCount: null,
+                  },
+                  fetching: false,
+                },
+              },
+              ui: {
+                nav: {
+                  foldersExpanded: false,
+                  visible: false,
+                },
+              },
+            },
+          },
+        },
+      };
+
+      view = renderInReduxProvider(<HealthCare dataLoadingDisabled />, {
+        initialState,
+        reducers,
+      });
+
+      await expect(view.queryByText(new RegExp(`Next appointment`, 'i'))).to
```

Comment:
should use `getBy` when you expect something to be there since it gives better error output should the element not exist. Also, if you want, when using `getBy` you don't even need to use `expect` since it'll throw an error. So you can just do `view.getByText(/next appointment/i)` here and if the element cannot be found, it'll fail :)

Reply:
Ah, good to know!